src/pages/Home.js

```diff
@@ -1,78 +1,108 @@
 import React from "react";
-import Grid from '@mui/material/Grid'
-import Container from '@mui/material/Container'
-import TextField from '@mui/material/TextField'
-import Button from '@mui/material/Button'
+import Grid from "@mui/material/Grid";
+import Container from "@mui/material/Container";
+import TextField from "@mui/material/TextField";
+import Button from "@mui/material/Button";
 
 export const Home = (e) => {
-    const zipCode = '33810'
-    const pageNumber = 0; //Math.floor(Math.random() * 11)
-    const restaurantsArrayNumber = Math.floor(Math.random() * 11);
-    
+  const zipCode = "33810";
+  const pageNumber = 0; //Math.floor(Math.random() * 11)
+  const restaurantsArrayNumber = Math.floor(Math.random() * 11);
 
-    const apiKey = process.env.REACT_APP_RESTAURANT_API_KEY
-    const options = {
-        method: 'GET',
-        headers: {
-            'X-RapidAPI-Key': apiKey,
-            'X-RapidAPI-Host': 'restaurants-near-me-usa.p.rapidapi.com'
-        }
-    };
-    
-    const restaurantsResults = async () => {
-        const results = await fetch(`https://restaurants-near-me-usa.p.rapidapi.com/restaurants/location/zipcode/${zipCode}/${pageNumber}`, options)
-        console.log(results.json)
-}
-restaurantsResults();
-    
-    return(
-        <Grid xs item={12} sx={{
-            background: 'gray',
-            padding: 0,
-            margin: 0,
-            height: '100%'
-        }}>
-            <Grid xs item={12}>
-                <h1 style={{
-                    textAlign: 'center',
-                    margin: 0
-                }}>Idk U Pick</h1>
+  const apiKey = process.env.REACT_APP_RESTAURANT_API_KEY;
+  const options = {
+    method: "GET",
+    headers: {
+      "X-RapidAPI-Key": apiKey,
+      "X-RapidAPI-Host": "restaurants-near-me-usa.p.rapidapi.com",
+    },
+  };
+
+  const restaurantsResults = async () => {
+    const results = await fetch(
+      `https://restaurants-near-me-usa.p.rapidapi.com/restaurants/location/zipcode/${zipCode}/${pageNumber}`,
+      options
+    );
+    console.log(results.json);
+  };
+  restaurantsResults();
+
+  return (
+    <Grid
+      xs
+      item={12}
+      sx={{
+        backgroundImage:
```

Comment:
Instead of doing inline styles lets make the react component a little easier to read by adding this package https://www.npmjs.com/package/styled-components

If we create styled components it should be easier to manage the JSX.

Reply:
Sounds good. I'll start working on that. 😁

Reply:
Correction, I'll start reading, learning, and watching about it... then I'll get started LOL.

Reply:
So when you say styled components, do you mean like this? [https://www.youtube.com/watch?v=-FZzPHSLauc](https://www.youtube.com/watch?v=-FZzPHSLauc)

If so, I think I could tackle this after getting my head around the overall structure of the app. I'm unfamiliar with the mui component library so that's next on my list to learn about since it is already in use in the project.

Reply:
Yes exactly like that. Also you can resolve the merge issue as practice. 
That will come up more often than not.